man/fi_endpoint.3.md

```diff
@@ -616,19 +616,20 @@ if data ordering is not maintained, message ordering may be.
 : Read after write size.  If set, an RMA or atomic read operation
   issued after an RMA or atomic write operation, both of which are
   smaller than the size, will be ordered.  The RMA or atomic read
-  operation will see the results of the previous RMA or atomic write.
+  operation will see the results of the previous RMA or atomic write
+  at the target memory location.
```

Comment:
Yup, this works great. Do you want me to update this PR? Can I use this wording for the other parts that I modified with 'location'?